jsapp/js/components/common/actionIcon.stories.tsx

```diff
@@ -111,3 +80,22 @@ export const AllIconStyles = () => (
     )}
   </div>
 )
+
+const testId = 'ActionIcon-click-test'
+export const TestClick: Story = {
+  args: {
+    variant: 'filled',
+    size: 'md',
+    iconName: 'edit',
+    onClick: fn(),
+    'data-testid': testId,
+  },
+  play: async ({ args, canvasElement }) => {
+    // Unfortunately Storybook doesn't pass proper types for `args`, so we need to cast it.
+    // TODO: I made an issue to point this out to Storybook team: https://github.com/storybookjs/storybook/issues/31106
+    // let's fix this when they fix it.
+    const canvas = within(canvasElement)
+    await userEvent.click(canvas.getByTestId(testId))
```

Comment:
this looks more scalable to me in case of more than single test case.
```suggestion
    await userEvent.click(canvas.getByTestId((args as StoryArgs)['data-testid']!))
```

Reply:
Sure :)